src/database.ts

```diff
@@ -976,6 +1017,59 @@ class Database extends GrpcServiceObject {
       callback!
     );
   }
+
+  /**
+   * Retrieves the restore information of the database.
+   *
+   * @see {@link #getMetadata}
+   *
+   * @method Database#getRestoreInfo
+   * @returns {Promise<IRestoreInfoTranslatedEnum | undefined>}
```

Comment:
Will users be able to click "IRestoreInfoTranslatedEnum" on the docs site and see what that means?

Reply:
I actually have no idea how to document this. It's a locally defined type in this file (not exported). I created a @typedef and hopefully that gets picked up by the docs.

Reply:
It worked! (you can preview how the docs turn out with "npm run docs" and then open up `docs/index.html`)